### PR TITLE
Fix the set_rabbitmq_policy.sh usage

### PR DESCRIPTION
**Pull request by bogdando**
* Call it only if exists
* Address packaging:
  - for stable branch, use packaging/common/ to contain the
    set_rabbitmq_policy.sh and include it to the debian rules
    and rpm spec
  
*NOTE*: for master branch landing please update as appropriate,
  as it should be using ./scripts source!

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

### CODE REVIEW

**Comment by bogdando** [comment]
Please don't merge w/o @dmitrymex review

**Comment by bogdando** [comment]
related to https://github.com/rabbitmq/rabbitmq-server/pull/480

**Comment by michaelklishin** [comment]
@bogdando @dmitrymex keep in mind that we plan on shipping `3.5.7` next Monday.

File: packaging/debs/Debian/debian/rules [hunk 1]
```diff
@@ -19,6 +19,7 @@ install/rabbitmq-server::
 	done
 	sed -e 's|@RABBIT_LIB@|/usr/lib/rabbitmq/lib/rabbitmq_server-$(DEB_UPSTREAM_VERSION)|g' <debian/postrm.in >debian/postrm
 	install -p -D -m 0755 debian/rabbitmq-server.ocf $(DEB_DESTDIR)usr/lib/ocf/resource.d/rabbitmq/rabbitmq-server
+	install -p -D -m 0755 debian/rabbitmq-server.ocf $(DEB_DESTDIR)usr/lib/ocf/resource.d/rabbitmq/set_rabbitmq_policy.sh
```
Comment (dmitrymex) [inline packaging/debs/Debian/debian/rules]: Here
debian/rabbitmq-server.ocf 
should be
debian/set_rabbitmq_policy.sh

**Comment by dmitrymex** [comment]
@bogdando: +1 from me, thank you. @michaelklishin: could you please merge this?

**Comment by dmitrymex** [comment]
@michaelklishin: thank you

**Comment by dumbbell** [comment]
FYI, I submitted #495 as an improvement to the Debian and RPM packages.

**Pull request merged**